phylogenetic/build-configs/ci/config.yaml

```diff
@@ -1,2 +1,5 @@
+serotypes: ["all"]
+genes: ["genome", "E"]
```

Comment:
Reduced CI runtime from [8m25s](https://github.com/nextstrain/dengue/actions/runs/18178926044) to [2m46s](https://github.com/nextstrain/dengue/actions/runs/18199646554). 

---
As a side note, this also prompted me to look into the CI runtime history and I noticed the CI used to complete in ~4m but was bumped up to ~8m with the [multi-threading support](https://github.com/nextstrain/dengue/pull/105). We could revisit making the threads more dynamic instead of hardcoding to max 8.